bin/tap-difflet

```diff
@@ -154,4 +158,13 @@ process.on('exit', function () {
   process.exit(errors.length);
 });
 
+function summarizeErrors(errors) {
+  var summary = chalk.underline.bold('\n\nTests that failed:\n');
+
+  _.each(errors, function(error) {
```

Comment:
You could use `errors.forEach` and not need to add lodash as a dep, unless we're concerned about supporting really old versions of Node.